resources/views/moderator/schedule/presentations/remove-presentation-from-schedule.blade.php

```diff
@@ -0,0 +1,11 @@
+<div>
+    <div class="font-semibold pb-2">
+        Remove presentation
+    </div>
+    <p class="text-md pb-1">If you want you cannot replace the presentation, but you want it off the schedule, you can
```

Comment:
Question: Is there a way to replace the presentation from this page? The grammar seems off.
```suggestion
    <p class="text-md pb-1">If you choose not to replace the presentation, you can just take it off the schedule.</p>
```